templates/memcached/config/memcached

```diff
@@ -3,5 +3,5 @@ USER="memcached"
 MAXCONN="8192"
 CACHESIZE="9932"
 # explicit IP to bind to (wrap IPv6 in brackets)
-LISTEN=$(echo 127.0.0.1 ::1 $POD_IPS | tr ' ' '\n' | sed 's/\(.*\):\(.*\)/[\1:\2]/' {{ .memcachedTLSListen }})
+LISTEN=$(echo 127.0.0.1 ::1 $(printf %s $POD_IPS | tr ',' ' ') | tr ' ' '\n' | sed 's/\(.*\):\(.*\)/[\1:\2]/' {{ .memcachedTLSListen }})
```

Comment:
The printf here is probably not necessary, as we only have the localhost addresses before $POD_IPS and we don't plan on adding more hard coded values here.
What about just keeping too tr sequentially instead? It should be sufficient
```
echo 127.0.0.1 ::1 $POD_IPS | tr ',' ' ' | tr ' ' '\n'
```

Reply:
works for me. will submit the change